src/main/java/com/softserve/kickscooter/kickscootervehicle/management/controller/ManagementController.java

```diff
@@ -0,0 +1,55 @@
+package com.softserve.kickscooter.kickscootervehicle.management.controller;
+
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterTechInfoDto;
+import com.softserve.kickscooter.kickscootervehicle.management.model.Scooter;
+import com.softserve.kickscooter.kickscootervehicle.management.service.ManagementService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+import java.util.UUID;
+
+@Slf4j
+@RestController
+@RequestMapping("/scooters")
+@AllArgsConstructor
+public class ManagementController {
+
+    private ManagementService service;
+
+    @GetMapping
+    public ResponseEntity<Iterable<ScooterTechInfoDto>> getAllScooterTechInfo(){
```

Comment:
user may want to get ordered results, Iterable doesn't guarantee order. it can be a separate task

Reply:
I understood